docs/content/manual/release-specific/v1.1.0/rwx_feature.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Test Read Write Many Feature
+---
+
+# Prerequisite:
+1. Set up a Cluster of 4 nodes (1 etc/control plane and 3 workers)
+2. Deploy Latest Longhorn-master
+
+# Create StatefulSet/Deployment with single pod with volume attached in RWX mode.
+1. Create a PVC with RWX mode using longhorn class by selecting the option `read write many`.
+2. Attach the PVC to a StatefulSet/Deployment with 1 pod.
+3. Verify that a PVC, ShareManger pod, CRD and volume in Longhorn get created.
+4. Verify share-manager pod is ready by checking the logs of the share-manager pod which should signal once the volume is attached, mounted and the nfs-server is up.
```

Comment:
don't need to check the logs, you can just check the container ready flag. (i.e. pod will show as ready in rancher)

Reply:
Corrected.